infrastructure/code/code.go

```diff
@@ -488,14 +479,29 @@ func (sc *Scanner) createBundle(
 		if !(len(fileContent) > 0 && len(fileContent) <= maxFileSize) {
 			continue
 		}
-		file := getFileFrom(filePath, fileContent)
-		bundleFiles[filePath] = file
-		fileHashes[filePath] = file.Hash
+		bundleFile := getFileFrom(filePath, fileContent)
+		bundleFiles[filePath] = bundleFile
+		fileHashes[filePath] = bundleFile.Hash
+		fileName := filepath.Base(filePath)
```

Comment:
this is most likely not correct - it should be the path from folder root - e.g. if you have several `index.js` it wouldn't work, if I understand correctly.